tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -175,7 +178,7 @@ public TiTableInfo copyTableWithRowId() {
           getId(), CIStr.newCIStr(getName()), getCharset(), getCollate(),
           true, newColumns.build(), getIndices(),
           getComment(), getAutoIncId(), getMaxColumnId(),
-          getMaxIndexId(), getOldSchemaId());
+          getMaxIndexId(), getOldSchemaId(), partitionInfo);
```

Comment:
Should we add a getter method for partitionInfo?

Reply:
It will be added later when I really need to use it.